src/v1/waittimes/db.ts

```diff
@@ -0,0 +1,19 @@
+import { print } from 'util';
+// Unused right now
+import { WaitTimeDocument } from './models/waittimes';
+import { firebaseDB } from '../auth';
+
+export class WaitTimeDB {
+
+  async waitTime(facilityId?: string) {
+    if (facilityId) {
+      return (
+        await firebaseDB.collection('waittimes').doc('waittimes').get()
+      ).get(facilityId);
+    }
```

Comment:
Nice job! Since our change in structure for the feedback, is the model going to reflect the predicted wait times at each eatery for certain days and times, or is it just storing an overall wait time? If we were to store predicted wait times based on day and hour, we might need to modify the existing structure of the collection a bit. Great work though!